README.md

```diff
@@ -4,13 +4,14 @@
 This is the simplest configuration for developers to start with.
 
 ### Initial Setup
-1. Run `docker-compose run --rm django ./manage.py migrate`
-2. Run `docker-compose run --rm django ./manage.py createsuperuser`
+1. Run `docker-compose run --rm npm npm install`
```

Comment:
Should there be `npm npm` (npm repeated twice)?

Reply:
yes, @zachmullen explained that the first `npm` is the service and `npm install` is the command to run:
https://docs.docker.com/compose/reference/run/